### DISCUSSION TITLE
Calculating area under the survival curve.

### DISCUSSION THREAD

**Discussion by guilhermelac001** [CamDavidsonPilon/lifelines#1406] [Q&A] [answered]
Hi! I am doing a project where I need to know the area under the curve between 2 points in time for a survival curve. I know that restricted_mean_survival_time can compute that for 0 to an upper bound, but I didn't find any method that does the same operation with a lower and upper bound. Is there anything that does just that?

**Comment by CamDavidsonPilon** [comment, marked as answer]
I haven't checked this, but maybe try `restricted_mean_survival_time(t) - restricted_mean_survival_time(s)` for the area in the interval `[s,t]` - make sense?

**Reply by guilhermelac001** [reply]
It worked! Thanks
